c++/src/H5Cpp.h

```diff
@@ -48,14 +48,4 @@
 #include "H5File.h"
 #include "H5Library.h"
 
-/* Some C++ compilers do not have offsetof macro; define to bypass the problem
-   - BMR- -EIP- 2007/08/01
-*/
```

Comment:
This is no longer true. C++11 adds additional functionality on top of offset(), so it will always be present.